makefiles/Makefile.devhub-content-integration

```diff
@@ -57,4 +60,4 @@ next-gen-html-publish:
 
 next-gen-stage: ## Host online for review
 	mut-publish public ${STAGING_BUCKET} --prefix="${COMMIT_HASH}/${MUT_PREFIX}" --stage ${ARGS};
-	echo "Hosted at ${STAGING_URL}/${COMMIT_HASH}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/";
\ No newline at end of file
+	echo "Hosted at ${STAGING_URL}/${COMMIT_HASH}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/";
```

Comment:
could we remove the newline added to 63?

Reply:
For sure! I think my editor automatically added this.